Library/Formula/rem.rb

```diff
@@ -0,0 +1,15 @@
+class Rem < Formula
+  desc "Command-line tool to access OSX Reminders.app database"
+  homepage "https://github.com/kykim/rem"
+  url "https://github.com/kykim/rem/archive/20150618.tar.gz"
+  sha256 "e57173a26d2071692d72f3374e36444ad0b294c1284e3b28706ff3dbe38ce8af"
+
+  def install
+    xcodebuild
```

Comment:
Thank you for your kindly help, I'd add `depends_on :xcode => :build` and rebased up-to-date.

Thought that xcode must be already installed during home-brew install.  ;)